accel.js

```diff
@@ -20,6 +25,7 @@ var address = 0x68;
 function listen() {
     var i2c1 = i2c.openSync(1);
     var sensor = new MPU6050(i2c1, address);
+    var sensitivity = nconf.get('accelsensitivity');
```

Comment:
nit: since the config is part of the accelerator module, this could be simply 'sensitivity', which would be more readable.